index.rst

```diff
@@ -302,107 +305,156 @@ Is this case, we could have something like:
+As described :ref:`above <section-salobj>`, CSCs written with Salobj define a configuration schema (e.g. `ts_atdome <https://github.com/lsst-ts/ts_ATDome/blob/develop/python/lsst/ts/ATDome/config_schema.py>`__).
+The configuration schema currently contains default values for the configuration parameters which are loaded if the ``start`` command is sent with an empty ``configurationOverride`` attribute (the default value).
 Nevertheless, the values in the schema are seldom valid beyond a unit testing environment, which requires users to provide some kind of *operational defaults* or *default label*.
-One can see how this can cause confusion when operating the system since now "default" has two different meanings, e.g.; *schema default* and *operational default*.
+One can see how this can cause confusion when operating the system since "default" can be interpreted in two different ways, e.g.; *schema default* and *operational default*.
 Furthermore, it is usually enough to override a small subset of the *schema defaults* for operations.
 Therefore, to get a full set of applied configurations, users must look at two distinct repositories; the configuration repository (for the modified parameters) and the CSC repository (for the schema defaults).
 
 The proposal to improve this aspect of the system is:
 
-#.  Remove default values from the configuration schema.
+#.  Remove all default values from configuration schema definition in the CSC repository.
 
     - See this :download:`example schema <_static/ATSpectrograph_schema.yaml>` for the ATSpectrograph CSC.
+    - Unit tests will need to utilize configuration files stored in the `tests/data/config` directory as is done for the `ATDome CSC <https://github.com/lsst-ts/ts_ATDome/tree/develop/tests/data/config>`_. See `Salobj documentation <https://ts-salobj.lsst.io>`__) for more details.
+
+#.  In the configuration repository for the given CSC (e.g `ts_config_attcs <https://github.com/lsst-ts/ts_config_attcs>`_ for the ATDome) there shall be a ``_init.yaml`` file defining all values that are expected to be common to all sites and/or be relatively static in operations (we intentionally use "_init" instead of "_default").
+
+    - See this :download:`example _init.yaml <_static/_init.yaml>` for the ATSpectrograph CSC.
+    - This file is the first configuration file loaded by the CSC
+    - It is not possible to load the ``_init.yaml`` file (or any file with a ``_`` prefix) using the ``configurationOverride`` attribute
+    - Note that all CSCs having multiple algorithms, each with different required attributes, must have an initial set of defaults in this file.
+
+#.  Also in the configuration repository for the given CSC, when applicable, there will be a file corresponding to each site where the CSC is used (e.g. ``_summit.yaml, _ncsa.yaml, _base.yaml``).
```

Comment:
Please clearly explain that these are optional (if we agree -- as I think we do -- see the discussion a bit below).

I fear this whole section needs minor editing to accommodate that change.

Reply:
I like your improvement to this section..